ospf6d/ospf6_nssa.c

```diff
@@ -1246,6 +1246,9 @@ int ospf6_area_nssa_set(struct ospf6 *ospf6, struct ospf6_area *area)
 {
 
 	if (!IS_AREA_NSSA(area)) {
+		/* Disable stub first. */
+		ospf6_area_stub_unset(ospf6, area);
```

Comment:
Shouldn't we also add `ospf6_area_no_summary_unset` here?

Reply:
Indeed. Fixed as well.

Reply:
Have you added ospf6_area_no_summary_unset() as suggested by @idryzhov ? I don't see it in latest commit yet.